src/config/lang/english.go

```diff
@@ -645,6 +645,8 @@ const (
 	PkgValidateErrManifestNameLength      = "manifest %q exceed the maximum length of %d characters"
 	PkgValidateErrManifestNameNotUnique   = "manifest name %q is not unique"
 	PkgValidateErrPkgConstantPattern      = "provided value for constant %q does not match pattern %q"
+	PkgValidateErrChartReleaseNameEmpty   = "release name empty, unable to fallback to chart name"
+	PkgValidateErrChartReleaseNameInvalid = "invalid release name '%s': a DNS-1035 label must consist of lower case alphanumeric characters or '-', start with an alphabetic character, and end with an alphanumeric character"
```

Comment:
We are trying to phase out use of the language package so just put the error message in the code. Also don't use quotes in your error messages if you do not really have to.

Reply:
Noted! I assumed this would be fine as a private constant since it's not exposed outside the package. I've updated the error messages to avoid using quotes unless necessary, and moved them directly into the code as recommended. Thanks for the guidance!